packages/graphql-language-service-interface/src/fileUtils.ts

```diff
@@ -0,0 +1,57 @@
+function getExtension (pathParts: string[]): string | null {
+  return pathParts.pop() || null
+}
+
+function removeExtension  (filePath: string, pathParts: string[]): string {
+  // if there's a file extension
+  if (pathParts.length > 1) {
+    // pop the extension from the array
+    pathParts.pop() as string
```

Comment:
```suggestion
    pathParts.pop()
```

Reply:
This probably shouldn't mutate the argument.

Reply:
well `pop()` can return `null | string` so it gets mad if i dont do this